docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -5,42 +5,62 @@
+{anomaly-detect-cap} alert:: 
+  Checks if the {anomaly-job} results contain anomalies that match the rule 
+  conditions.
+
+{anomaly-jobs-cap} health:: 
+  Monitors job health and alerts if an operational issue occurred that may 
+  prevent the job from detecting anomalies.
+
+TIP: If you have created rules for specific {anomaly-jobs} and you want to 
+monitor whether these jobs work as expected, {anomaly-jobs} health rules are 
+ideal for this purpose.
+
+
+[[creating-ml-rules]]
 == Creating a rule
 
 You can create {ml} rules in the {anomaly-job} wizard after you start the job, 
-from the job list, or under **{stack-manage-app} > {alerts-ui}**. On the *Create 
-rule* window, select *{anomaly-detect-cap} alert* under the {ml} section, then 
-give a name to the rule and optionally provide tags.
-
-Specify the time interval for the rule to check detected anomalies. It is 
-recommended to select an interval that is close to the bucket span of the 
-associated job. You can also select a notification option by using the _Notify_ 
-selector. An alert remains active as long as anomalies are found for a 
-particular {anomaly-job} during the check interval. When there is no anomaly 
-found in the next interval, the `Recovered` action group is invoked and the 
-status of the alert changes to `OK`. For more details, refer to the 
-documentation of 
+from the job list, or under **{stack-manage-app} > {alerts-ui}**.
+
+On the *Create rule* window, give a name to the rule and optionally provide 
+tags. Specify the time interval for the rule to check detected anomalies or job 
+health changes. It is recommended to select an interval that is close to the 
+bucket span of the job. You can also select a notification option with the 
+_Notify_ selector. An alert remains active as long as the configured conditions 
+are met during the check interval. When there is no matching condition in the 
+next interval, the `Recovered` action group is invoked and the status of the 
+alert changes to `OK`. For more details, refer to the documentation of 
 {kibana-ref}/create-and-manage-rules.html#defining-rules-general-details[general rule details].
-  
+
+Select the rule type you want to create under the {ml} section and continue to 
+configure it depending on whether it is an 
+<<creating-anomaly-alert-rules, {anomaly-detect} alert>> or an 
+<<creating-anomaly-jobs-health-rules, {anomaly-job} health>> rule.
+
 [role="screenshot"]
-image::images/ml-anomaly-alert-type.jpg["Creating a rule for an anomaly detection alert"]
-  
-Select the {anomaly-job} or the group of {anomaly-jobs} that is checked against 
-the rule. If you assign additional jobs to the group, the new jobs are 
-automatically checked the next time the conditions are checked.
+image::images/ml-rule.jpg["Creating a new machine learning rule"]
+
+
+[[creating-anomaly-alert-rules]]
+=== {anomaly-detect-cap} alert
+
+Select the job or the group that the rule applies to. If you assign more jobs to 
```

Comment:
For anomaly detection alerting rule we only support single job selection at the moment